extensions/amp-reddit/0.1/test/test-amp-reddit.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  createIframePromise,
+  doNotLoadExternalResourcesInTest,
+} from '../../../../testing/iframe';
+import '../amp-reddit';
+import {adopt} from '../../../../src/runtime';
+import {reddit} from '../../../../3p/reddit';
+
+adopt(window);
+
+describe('amp-reddit', () => {
+  function getReddit(src, type) {
+    return createIframePromise().then(iframe => {
```

Comment:
`describes.realWin()` is preferred in new tests. example: https://github.com/ampproject/amphtml/blob/master/test/functional/test-extensions.js#L297